src/main/scala/reactive/tweets/incoming/TweetActorManager.scala

```diff
@@ -0,0 +1,26 @@
+package reactive.tweets.incoming
+
+import akka.actor.{ Actor, Props }
+import reactive.tweets.domain.{ Tweet, User }
+import reactive.tweets.incoming.TweetActor.GetLastTen
+
+class TweetActorManager extends Actor {
+
+  override def receive = {
+    case tweet: Tweet        => forward(tweet, tweet.user.name)
```

Comment:
In de applicatie zou Ordina naar ordina-site moeten linken en de menu-items hebben geen implementatie dus aub verwijderen